bittensor/_dendrite/__init__.py

```diff
@@ -41,7 +41,7 @@ def __new__(
             max_worker_threads: int = None,
             max_active_receptors: int = None,
             receptor_pool: 'bittensor.ReceptorPool' = None,
-            multiprocess: bool = False,
+            multiprocess: bool = None,
```

Comment:
Should Bool have a default value of `None`?

Reply:
hmmm this is for the